sceptre/cli/helpers.py

```diff
@@ -134,6 +134,8 @@ def _generate_yaml(stream):
             [yaml.load(item, Loader=CfnYamlLoader) for item in items],
             default_flow_style=False, explicit_start=True
         )
+    elif isinstance(stream, dict):
+        return yaml.dump(stream, default_flow_style=False)
```

Comment:
why did you omit the "explicit_start" argument that is used elsewhere in this function?

Reply:
Nope, I just didn't notice exlicit start was used elsewhere.